src/components/events/finance-marketing.tsx

```
@@ -1,21 +1,643 @@
+              viewport={{ once: true }}
+            >
+              <Card
+                className={cn(
+                  "bg-red-800/30 border-red-700/50 transition-all cursor-pointer backdrop-blur-sm",
+                  openIndex === index && "bg-red-800/50 border-red-600/60"
+                )}
+              >
+                <button
+                  onClick={() => setOpenIndex(openIndex === index ? null : index)}
+                  className="w-full p-6 text-left flex items-start justify-between gap-4"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)